packaging/installer/methods/packages.md

```diff
@@ -2,46 +2,78 @@
+architectures), these packages integrate tightly with your system's package manager, making them easier to maintain,
+update, and uninstall.
 
-Netdata is committed to support installation of our solution to all operating systems. This is a constant battle for
-Netdata, as we strive to automate and make things easier for our users. For the operating system support matrix, please
-visit our [distributions](../../DISTRIBUTIONS.md) support page.
+We currently use [packagecloud](https://packagecloud.io/netdata/) to supply repositories and packages. We provide two
+separate repositories, one for nightly releases, and another for stable releases.
 
-We provide two separate repositories, one for our stable releases and one for our nightly releases.
+-   Nightly releases: [netdata/netdata-edge](https://packagecloud.io/netdata/netdata-edge)
+-   Stable releases: [netdata/netdata](https://packagecloud.io/netdata/netdata)
 
-1.  Stable releases: Our stable production releases are hosted in
-    [netdata/netdata](https://packagecloud.io/netdata/netdata) repository of package cloud
-2.  Nightly releases: Our latest releases are hosted in
-    [netdata/netdata-edge](https://packagecloud.io/netdata/netdata-edge) repository of package cloud
+Read our notice about [nightly vs. stable releases](../README.md#nightly-vs-stable-releases) to understand the
+differences between the two.
 
-Visit the repository pages and follow the quick set-up instructions to get started.
+## Quickstart
 
-## Using caching proxies with PackageCloud repositories
+packagecloud offers helper installation scripts to add their repositories to your package manager. Jump to your
+distribution and use the commands to install Netdata get _automatic nightly updates_ via your distribution's package
+manager. You can also manually add the repositories on [`.deb`
+systems](https://packagecloud.io/netdata/netdata-edge/install#manual-deb) or [`.rpm`
+systems](https://packagecloud.io/netdata/netdata-edge/install#manual-rpm).
 
-PackageCloud only provides HTTPS access to repositories they host, which
-means in turn that Netdata's package repositories are only accessible
-via HTTPS. This is known to cause issues with some setups that use a
-caching proxy for package downloads.
+-   [Ubuntu and Debian (`.deb`)](#ubuntu-and-debian-deb)
+-   [Fedora (`.rpm`)](#fedora-rpm)
+
+### Ubuntu and Debian (`.deb`)
+
+Use the helper script to add the packagecloud repository, then use `apt` to install Netdata.
+
+```bash
+curl -s https://packagecloud.io/install/repositories/netdata/netdata-edge/script.deb.sh | sudo bash
+sudo apt-get update
+sudo apt-get install netdata
+```
+
+Skip ahead to the [What's next?](#whats-next) section to find links to helpful post-installation guides.
+
+### Fedora (`.rpm`)
+
+Use the helper script to add the packagecloud repository, then use `dnf` to install Netdata.
+
+For `.rpm` systems using YUM (Fedora, CentOS, RHEL, OpenSuSE):
```

Comment:
We shouldn't be listing OpenSUSE here, it will need it's own section (the setup script works fine for it, but SUSE doesn't use YUM/DNF, they use either YaST or Zypper).